src/index.html

```diff
@@ -9,6 +9,26 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Enclosures</h1>
+  <div class="big" id="qa-block">
+    <div class="small">
+      <div class="smallest">
```

Comment:
that are not good options for names
`outer-box`, `middle-box`, `inner-box` are better